packages/atdpy/atdpy.2.3.2/opam

```diff
@@ -0,0 +1,59 @@
+opam-version: "2.0"
+synopsis: "Python/mypy code generation for ATD APIs"
+description: "Python/mypy code generation for ATD APIs"
+maintainer: [
+  "Louis Roché <louis@louisroche.net>"
+  "Martin Jambon <martin@mjambon.com>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+]
+authors: [
+  "Martin Jambon <martin@mjambon.com>"
+  "David Sheets <sheets@alum.mit.edu>"
+  "Rudi Grinberg <me@rgrinberg.com>"
+  "Ivan Jager <aij+git@mrph.org>"
+  "Jeff Meister <nanaki@gmail.com>"
+  "Carmelo Piccione <carmelo.piccione@gmail.com>"
+  "Raman Varabets <roman.vorobets@gmail.com>"
+  "Mathieu Baudet <mathieubaudet@fb.com>"
+  "Rauan Mayemir <rauan@mayemir.io>"
+  "Louis Roché <louis@louisroche.net>"
+  "Brendan Long <self@brendanlong.com>"
+  "Christophe Troestler <christophe.Troestler@umons.ac.be>"
+  "Vincent Bernardoff <vb@luminar.eu.org>"
+  "haoyang <haoyang@esper.co>"
+]
+license: "MIT"
+homepage: "https://github.com/ahrefs/atd"
+bug-reports: "https://github.com/ahrefs/atd/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "1.11"}
+  "alcotest" {with-test}
+  "atd" {>= "2.3.0"}
+  "cmdliner" {>= "1.1.0"}
```

Comment:
```suggestion
  "atdgen" {>= "2.3.0"}
  "cmdliner" {>= "1.1.0"}
```

`atdpy` depends on `atdgen_emit` which is a private library that requires yojson and seems to be part of atdgen:
```
#=== ERROR while compiling atdpy.2.3.2 ========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/ahrefs/atd/releases/download/2.3.2/atdgen-codec-runtime-2.3.2.tbz)
# path                 ~/.opam/4.08/.opam-switch/build/atdpy.2.3.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p atdpy -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/atdpy-4616-761d80.env
# output-file          ~/.opam/log/atdpy-4616-761d80.out
### output ###
# File "atdgen/src/dune", line 5, characters 38-44:
# 5 |  (libraries atd biniou easy-format re yojson))
#                                           ^^^^^^
# Error: Library "yojson" not found.
# -> required by library "atdgen_emit" in _build/default/atdgen/src
# -> required by executable Atdpy_main in atdpy/src/bin/dune:2
# -> required by _build/default/atdpy/src/bin/Atdpy_main.exe
# -> required by _build/install/default/bin/atdpy
# -> required by _build/default/atdpy.install
# -> required by alias install
```